src/main/java/uk/co/ramp/covid/simulation/lockdown/LockdownController.java

```diff
@@ -1,36 +1,30 @@
 package uk.co.ramp.covid.simulation.lockdown;
 
 import uk.co.ramp.covid.simulation.util.Time;
-import uk.co.ramp.covid.simulation.population.Population;
-
 import java.util.ArrayList;
 import java.util.List;
 
 public class LockdownController {
-    private final Population population;
-    private List<LockdownEvent> events;
-    private List<LockdownEventGenerator> eventGenerators;
+    private final List<LockdownEvent> events;
+    private final List<LockdownEventGenerator> eventGenerators;
 
-    public LockdownController(Population p) {
-        population = p;
+    public LockdownController() {
         events = new ArrayList<>();
         eventGenerators = new ArrayList<>();
     }
     
-    public LockdownController addComponent(LockdownEvent c) {
+    public void addComponent(LockdownEvent c) {
```

Comment:
The original idea of this was so that we could initialise lockdowns like:

addComponent(New lockdown()).addComponent(New otherLockdown()).add ...

But if it doesn't look like we are using it (even in the tests) I'm happy for this to be removed (really just making this comment as a note in case we want to revert to that for whatever reason later!)

Reply:
It's easy to add it back when we want it.